cmd_actuators_mux/README.md

```diff
@@ -4,11 +4,26 @@ It also deactivate the current allowed topic if no messages are received after a
 
 ![cmd mux diagram](doc/cmd_multiplexer_diagram.png "Diagram")
 
-The multiplexer node publishes which input is in control on the topic "active":
-```
-/CmdActuatorMuxNodelet/fin_angle_active
-/CmdActuatorMuxNodelet/set_rpm_active
-```
+### Subscribed Topics
+
+*/input/autopilot/set_angles* ([fin_control/SetAngles](../fin_control/msg/SetAngles.msg): Receives from the `autopilot` node the fins angle position.
+
+*/input/autopilot/set_rpm* ([thruster_control/SetRPM](../thruster_control/msg/SetRPM.msg): Receives from the `autopilot` node the Thruster velocity.
```

Comment:
@patapufate why do we capitalize Thruster?
```suggestion
*/input/autopilot/set_rpm* ([thruster_control/SetRPM](../thruster_control/msg/SetRPM.msg): Receives from the `autopilot` node the thruster velocity.
```

Reply:
Also, nit^N: it receives a speed, not a velocity.

Reply:
Fixed